python/LIDarkNews.py

```diff
@@ -374,43 +435,32 @@ def SaveInterpolationTables(self, total=True, diff=True):
             ) as f:
                 np.save(f, self.differential_cross_section_table)
 
-    ##### START METHODS FOR SERIALIZATION #########
-    # def get_initialized_dict(config):
-    #     # do the intitialization step
-    #     pddn = PyDerivedDarkNews(config)
-    #     return pddn.__dict__
-    #     # return the conent of __dict__ for PyDerivedDarkNews
-
-    # @staticmethod
-    # def get_config(self):
-    #     return self.config
-    ##### END METHODS FOR SERIALIZATION #########
-
     def GetPossiblePrimaries(self):
         return [Particle.ParticleType(self.ups_case.nu_projectile.pdgid)]
 
```

Comment:
Dumb suggestion, but you could encapsulate this in a function.

```suggestion

    def _ensure_configured(self):
        if not self.is_configured:
            self.configure()
```